readme.md

```diff
@@ -30,16 +30,27 @@ The output is a JSON object in the form:
 
 ```json
 {
-    "bundle_name": {
-        "asset_kind": "/public/path/to/asset"
-    }
+    "chunks": {
+      "bundle_name": {
+          "asset_kind": "/public/path/to/asset"
+      }
+    },
+    "images": [{
+        "name": "hello.svg",
+        "path": "/public/path/to/asset/hello.svg"
+      }
+    ]
```

Comment:
If you wish, you can finalize it (it's required to add an option to give user an ability to override `/\.(jpe?g|png|gif|svg)$/`).